browser/ephemeral_storage/blob_url_browsertest.cc

```diff
@@ -71,11 +71,6 @@ class BlobUrlBrowserTestBase : public EphemeralStorageBrowserTest {
   };
   using FramesWithRegisteredBlobs = std::vector<RenderFrameHostBlobData>;
 
-  void SetUpOnMainThread() override {
-    EphemeralStorageBrowserTest::SetUpOnMainThread();
-    ASSERT_TRUE(embedded_test_server()->Start());
```

Comment:
i don't follow why this is moved

Reply:
it is already called in `EphemeralStorageBrowserTest::SetUpOnMainThread`. I had to resolve several issues across tests that subclass `EphemeralStorageBrowserTest` so I moved it there to simplify things.

Reply:
I don't recall the exact problem now, but I think maybe it needed be called before https://github.com/brave/brave-core/pull/23657/files/7091e291ca6d99689b310ff5aac503373442412b#diff-ad1b5bc543791a83f0e762f32a01e88544b826f9ecdad47ce2b8a49461cf7a7bR192